package.json

```diff
@@ -15,23 +16,33 @@
   "author": "Brian Leathem",
   "license": "MIT",
   "dependencies": {
+    "connect-mongo": "^1.3.2",
+    "connect-redis": "^3.1.0",
     "express": "4.14.0",
+    "express-session": "^1.14.2",
     "fh-mbaas-api": "5.14.2",
+    "fh-wfm-mediator": "0.1.0-rc1",
```

Comment:
This version deprecates the array-store in the mediator in favor of simple-store, but it's already outside the session management feature...

It was added here https://github.com/feedhenry-raincatcher/raincatcher-user/pull/27